dashboard/test/controllers/reference_guides_controller_test.rb

```diff
@@ -0,0 +1,31 @@
+require 'test_helper'
+
+class ReferenceGuidesControllerTest < ActionController::TestCase
+  include Devise::Test::ControllerHelpers
+
+  setup do
+    File.stubs(:write)
+    Rails.application.config.stubs(:levelbuilder_mode).returns true
+    @levelbuilder = create :levelbuilder
+    unit_group = create :unit_group, name: 'fake-course-2021'
+    course_offering = create :course_offering, key: 'fake-course'
```

Comment:
You can create course offering and course version by calling `CourseOffering.add_course_offering(unit_group)` and that will make sure it is set up correctly for the unit_group